Readme.md

```diff
@@ -80,3 +78,86 @@ Both modes support the following options:
     ```
     ./factory login --udp 1053:53
     ```
+
+* `--vnc`: open a VNC server. The port number must be between 5900 and 5999.
+
+    ```
+    ./factory login --vnc 5900
+    ```
+
+* `--commit`: after the VM quits, save the changes to its disk.
```

Comment:
You're saying this is interactive (asks for confirmation when the VM dies before overwriting). Where's the `-y` or `--always-commit` or something? Also, document it here.